frontend/src/redux/reducers/costEstimateReducer.js

```diff
@@ -14,6 +15,17 @@ const costEstimateView = (state = defaultCostEstimate, action) => {
   }
 };
 
+const questionList = (state = defaultQuestionList, action) => {
+  switch (action.type) {
+    case TOGGLE_LOGIC: {
```

Comment:
the TOGGLE_LOGIC is used for toggling the logic panel, so we'll need a new action created and used here!